packages/core/src/utils/editor.ts

```diff
@@ -41,6 +52,13 @@ export function checkHasEditorType(editor: EditorType): boolean {
   const commandConfig = editorCommands[editor];
   const command =
     process.platform === 'win32' ? commandConfig.win32 : commandConfig.default;
+
+  // Special handling for VS Code on macOS
+  if (editor === 'vscode' && process.platform === 'darwin') {
```

Comment:
Instead of this special casing would it be possible to build in the fallback logic into the editorCommands declaration?

Reply:
Hi, I think that's a nice idea would make more sense and code would be cleaner, I will implement this and update the code soon.